docs/develop-branch.md

```diff
@@ -0,0 +1,37 @@
+develop branch
+===
+
+Building the full Windows App SDK package involves multiple repos and build pipelines.
+Beyond the publicly-visible repos, we also have closed-source repos that produce public packages which
+are then consumed by this repo ([WindowsAppSDK](https://github.com/microsoft/WindowsAppSDK)).
+
+Some of those packages can't yet be pushed to the public feed on a CI/nightly basis though.
+Until that's resolved, we could get this repo into a state of being buildable only internally,
+by restoring the latest package from an internal feed.
+
+To allow either configuration to be built (using the public dependent package or the internal one),
+we maintain two branches (similar to the GitFlow process):
+
+* `main` branch consumes the latest public package
+* `develop` branch consumes the latest internal package.
```

Comment:
Still not a huge fan of calling this "buildable only by Microsoft people" branch "develop." I don't have a better name, though.

Reply:
The goal was to be up front that we're basically using GitFlow here, which uses the name 'develop'.